.github/workflows/php-security.yml

```
@@ -0,0 +1,53 @@
+name: PHP Security Checker
+
+on:
+  pull_request:
+
+jobs:
+  php-security-checker:
+    name: PHP Security Checker
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "locked"
+        php-version:
+          - "7.4"
```

Only for testing, we can try different dependencies for security
```suggestion
      matrix:
        dependencies:
          - "lowest"
          - "highest"
          - "locked"
        php-version:
          - "7.4"
          - "8.0"
```